omnibus/omnibus.rb

```diff
@@ -35,7 +35,7 @@
 
 # Enable S3 asset caching
 # ------------------------------
-use_s3_caching true
+use_s3_caching false 
```

Comment:
This is only for merge into `unified_workflow_spike`, which isn't yet open as a PR against master.